.github/workflows/test.yml

```diff
@@ -20,8 +20,9 @@ jobs:
     strategy:
       matrix:
         node-version:
-          - 18.0.0
-          - 19
+          - 18.17.0
+          - 20.6.0
+          - 20
```

Comment:
this enables testing the upper bound of the v20 major. i did not include similar for the v18 range since the risk of breaking that upper bound is lower since we are also testing in v20